cl/sentinel/gossip.go

```diff
@@ -495,6 +494,24 @@ func (g *GossipManager) Close() {
 	})
 }
 
+func (g *GossipManager) Start(ctx context.Context) {
```

Comment:
I feel the naming could be more clear like `MonitorTopicSubscriptions` or anything else

Reply:
Let's leave this for another PR